From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20Offenh=C3=A4user?= <offenhaeuser@protonmail.com>
Date: Sat, 11 Feb 2023 00:05:55 +0100
Subject: [PATCH] Hard-code some paths for runtime purposes

GemRB uses the paths where its libraries and data are copied to during
installation at runtime by generating a header. This does not work for
us, as our paths change from compilation to runtime. The easiest fix for
this is to hard-code these paths in the header file.
---
 cmake/cmake_config.h.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/cmake_config.h.in b/cmake/cmake_config.h.in
index b42a954c6f86fb3757b9d5f6fff055edcb831d13..a80a11a8dbe127e149144cae16cebca92f138820 100644
--- a/cmake/cmake_config.h.in
+++ b/cmake/cmake_config.h.in
@@ -8,9 +8,9 @@
 #cmakedefine HAVE_MEMALIGN 1
 #cmakedefine HAVE_ALIGNED_MALLOC 1
 #cmakedefine HAVE_POSIX_MEMALIGN 1
-#cmakedefine PLUGIN_DIR "${PLUGIN_DIR}"
-#cmakedefine DATA_DIR "${DATA_DIR}"
-#cmakedefine SYSCONF_DIR "${SYSCONF_DIR}"
+#cmakedefine PLUGIN_DIR "/usr/local/lib/gemrb/plugins/"
+#cmakedefine DATA_DIR "/usr/local/share/gemrb/"
+#cmakedefine SYSCONF_DIR "/usr/local/etc/gemrb/"
 #cmakedefine OPENGL_BACKEND ${OPENGL_BACKEND}
 #cmakedefine NOFPSLIMIT ${NOFPSLIMIT}
 #cmakedefine HAVE_UNISTD_H 1
